parts/openshift/openshiftnodescript.sh

```diff
@@ -23,3 +23,8 @@ update-ca-trust
 # note: ${SERVICE_TYPE}-node crash loops until master is up
 systemctl enable ${SERVICE_TYPE}-node.service
 systemctl start ${SERVICE_TYPE}-node.service &
+
+while [[ $(KUBECONFIG=/etc/origin/node/node.kubeconfig oc get node $(hostname) -o template \
```

Comment:
It's ok for a node to be unready as the system comes up, why wait?

Reply:
it has the advantage that the scale up call doesn't return success until all the requested nodes have successfully joined the cluster, and that if any new node fails to register, the error is reported as a deployment failure

Reply:
Is there any timeout used during scale up?

Reply:
same as on deploy - arm api will timeout, not sure exactly after how long - something like 45 minutes?